packages/aws-cdk/lib/init-templates/app/python/README.md

```diff
@@ -0,0 +1,42 @@
+
+Welcome to your CDK Python project!
```

Comment:
This template should actually be called "sample-app". The "app" template should result in an empty stack (so you don't have to go delete things). See typescript as an example.

Reply:
I'm confused by this comment.  I was following the existing examples for `csharp`, `java`, etc.  Is this inconsistent with them?

Reply:
Well.... those should be changed to ;-)

Reply:
Ah, I see.  Yeah, its kind of backwards now.   Do you want to fix all of them now?

Reply:
So do you want to change all of those now?

Reply:
Let's start by renaming this specific template to `sample-app` and then follow up with a PR for an empty `app` in python.